RealmSwift/LinkingObjects.swift

```diff
@@ -318,9 +318,10 @@ import Realm
      - parameter block: The block to be called whenever a change occurs.
      - returns: A token which must be held for as long as you want updates to be delivered.
      */
-    public func observe(on queue: DispatchQueue? = nil,
+    public func observe(keyPaths: [String]? = nil,
```

Comment:
The doctoring needs updating for the keyPaths parameter.

Reply:
Should be updated in latest.